package.json

```diff
@@ -197,6 +197,7 @@
     "lint-fix": "eslint --ext .js,.ts,.tsx --max-warnings 0 --fix .",
     "postinstall": "patch-package --error-on-fail && jetify",
     "validate": "npm ls",
+    "tsc-test": "tsc --project tsconfig.native.json --listFilesOnly",
```

Comment:
I've used this before too, how about we add some `tsc-test:web` and `tsc-test:native` scripts?